docs/installation-getting-started/pieces-os.mdx

```diff
@@ -0,0 +1,150 @@
+  return <a target="_blank" style={{textDecoration: "none"}} href={ props.src }>
+    <div style={{maxWidth: "fit-content", minWidth: "fit-content"}}>
+      <p style={{
+        padding: "8px 20px",
+        borderRadius: "8px",
+        border: "1px solid #f3f5f6",
+        backgroundColor: "black",
+        color: "white",
+        fontWeight: "500",
+        marginBottom: "8px",
+        marginTop: "25px"
+      }}>{props.children}</p>
+    </div>
+  </a>
+};
+
+export const MiniSpacer = props => {
+  return <div style={{ height: "16px"}}></div>
+};
+
+export const CallToAction = props => {
+  return <div style={{
+    marginTop: "150px",
+    marginBottom: "150px",
+    padding: "32px",
+    border: "1px solid gray",
+    borderRadius: "16px",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "start"
+  }}>
+    {props.children}
+    <h3 style={{ marginTop: "0px", marginBottom: "0px"}}>{props.heading}</h3>
+    {props.text}
+  </div>
+};
+
+# What is Pieces OS?
+
+Pieces OS operates as a background service, functioning both offline and on-device, facilitating communication among individual products within the Pieces for Developers suite. Its core functions include the management of local, private, and secure machine learning models used for [Pieces Copilot](https://docs.pieces.app/features/pieces-copilot), as well as [real-time search](gttps://docs.pieces.app/features/search-modes) and suggestions.
+
+![Pieces Ecosystem](https://storage.googleapis.com/pieces_multimedia/ILLUSTRATIONS/PIECES_FOR_DEVELOPERS/ANY_INTEGRATION/ANY_THEME/PLUGINS/1X1/PIECES_FOR_DEVELOPERS-ANY_INTEGRATION-PLUGINS-ANY_THEME-1X1-10_16_2023.PNG)
+
+Serving as a helper service, Pieces OS enables Pieces products to operate 100% locally on your machine, with an option to connect to the cloud for [backup](https://docs.pieces.app/features/back-up-and-restore), synchronization, and [sharing](https://docs.pieces.app/features/one-click-snippet-sharing) purposes.
+
+## Core Features of Pieces OS
+
+### Data Processing
+
+Pieces OS orchestrates effective data processing workflows, thus maintaining data integrity and providing real-time insights. Its local operation ensures a seamless and efficient data handling process, fostering enhanced development workflows.
+
+### Machine Learning Models Management
+
+Pieces OS's capability extends to managing various machine learning models, either stored locally or in the cloud. It ensures optimal performance and updated models for your applications, bolstering the efficacy and reliability of the machine learning aspects of your development projects.
+
+### Secure and Private
+
+Security and privacy are paramount in Pieces OS's operation. By running locally on your machine, it offers a secure environment for data processing and machine learning model management, ensuring the confidentiality and integrity of your data.
+
+## Getting Started with Pieces OS
+
+### Installation Guide
```

Comment:
if we do keep this section, we can probably just put the OS direct download links in each of these tabs for Pieces OS